packages/app-bridge/src/react/useGuidelineAssetsSettings.ts

```diff
@@ -0,0 +1,27 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect, useState } from 'react';
+import { Handler } from 'mitt';
+
+import type { GuidelineAssetsSettings } from '../types/GuidelineAssetsSettings';
+
+export const useGuidelineAssetsSettings = () => {
```

Comment:
```suggestion
export const usePrivacySettings = () => {
```
wdyt?

Reply:
makes sense 👍

Reply:
i'll update the event name's accordingly and the getters in a followup PR in clarify